docs/userGuide/syntax/mathformulae.mbdf

```diff
@@ -0,0 +1,99 @@
+## Math Formulae
+Markbind supports typesetting TeX math equations. [KaTeX](https://katex.org) is used as a fast math renderer.
+
+<box type="info">
+
+Insert `inline` equations by enclosing them in round brackets `\( ... \)`.
+
+Insert `display` equations by enclosing them in square brackets `\[ ... \]`.
+
+Insert numbered `display` equations by enclosing the equation square brackets and the equation number in curly brackets `\[ ... \] (1)`.
+</box>
+
+<box type="important">
+
+When typing `display` equations, this does not work as well:
+
+```markdown
+abc
```

Comment:
https://github.com/MarkBind/markbind/pull/1335#discussion_r480207052

sorry if it was misinterpreted, I didn't mean for you to place this exactly in the docs, it was only to illustrate that it is intentional behaviour by the [commonmark spec](https://commonmark.org/)

I think we should remove _all_ problems related to the markdown spec(s) in general, and let the author get accustomed to it.
if not, just brief mentions that universally apply, like the one we have for blank lines here https://markbind.org/userGuide/usingHtmlJavaScriptCss.html, otherwise every section in our ug would be double in size =X

Reply:
ah I see, I'll remove this section instead and assume the reader has read the `usingHtmlJavaScriptCss.html` section